src/App.js

```diff
@@ -9,15 +9,15 @@ mapboxgl.accessToken = 'pk.eyJ1Ijoia2F2aXRhLW0iLCJhIjoiY2t6eHI1MDE5MDFxZzJ5czBkc
 function App() {
```

Comment:
you could create a Map component and then import it into App(), so app stays clean 🤔